server/ws_internal_methods.ts

```diff
@@ -47,7 +47,12 @@ function emit(
     dispatchEvent(
       new CustomEvent("emit", { detail: { method, params } }),
     );
-    return { event: "emitted", id, method };
+    return {
+      event: "emitted",
+      id,
+      method,
+      ...(typeof params !== "undefined" ? { params } : {}),
```

Comment:
I also added the `params` here to the emitted events, which were not used before. I think they are meant to be the event payload, so should be part of the RPC message, right?